README.md

```diff
@@ -7,11 +7,12 @@
 - Select executable installer from the options that appear.
 
 We used Python 3.6.4 to make our stuff.
+Also we used an IDE but Python comes with IDLE. Just saying IDE is better.
 
 ## Run the game
 
 - From the command prompt, enter the following command:
     ```
     cd invadedhouse
     python3 run.py
-    ```
\ No newline at end of file
+    ```
```

Comment:
didn't edit this. Did not. I mean look, its exactly the SAME!

Reply:
maybe the whitespace is a little different. either way it still works!